REQUIRE

```diff
@@ -1,4 +0,0 @@
-julia 1.0
```

Comment:
REQUIRE files are still needed for registered packages

Reply:
OK, I put it back